Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
```

Comment:
### [faraday](https://lostisland.github.io/faraday)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.7.11 → 2.8.1

[_[change-log](https://github.com/lostisland/faraday/releases/tag/v2.8.1), [source-code](https://github.com/lostisland/faraday), [gem-diff](https://my.diffend.io/gems/faraday/2.7.11/2.8.1)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/lostisland/faraday/compare/v2.7.11...v2.8.1).

These are the first 10 commits:
- (5fccc20) [Update parallel-requests.md](https://github.com/lostisland/faraday/commit/5fccc208549650b129b13a173af3f3f40d44c14d)
- (63ba2aa) [429 error](https://github.com/lostisland/faraday/commit/63ba2aaf0c6127d88ced4a7af3c2d5efac698812)
- (3e27447) [Version bump to 2.7.12](https://github.com/lostisland/faraday/commit/3e27447a8016b6fc42829c1b413d82a6d0ea2d77)
- (342e7e6) [Update testing.md](https://github.com/lostisland/faraday/commit/342e7e65cdf4195a8f2c95f072187ac2f47aa54b)
- (639e1b5) [Gemfile: add racc to development, lint group](https://github.com/lostisland/faraday/commit/639e1b52e76da6a3cb7366f014227df0a5ba12f6)
- (53c7b49) [Lint](https://github.com/lostisland/faraday/commit/53c7b499db87894c56d9a556ff9bddf5852b002d)
- (fa6075c) [lint: Style/HashEachMethods](https://github.com/lostisland/faraday/commit/fa6075c0f95f0b22ad82fd418d00b19046d2d949)
- (09a635f) [CI: tell dependabot to update GH Actions](https://github.com/lostisland/faraday/commit/09a635f30ef002c3f1eb5ba79a60aaeb4b401777)
- (b8e2e45) [Bump actions/checkout from 3 to 4](https://github.com/lostisland/faraday/commit/b8e2e454df954fe561b3a14169fcc9caf0abd695)
- (1e81e2c) [Configurable JSON encoders and decoders (#1539)](https://github.com/lostisland/faraday/commit/1e81e2c71a8b2ab8cc4b98a904a0edee333e2aa7)


</details>